README.md

```diff
@@ -125,13 +124,13 @@ Now when the container starts, all files in `/opt/dd-agent-conf.d` with a `.yaml
 
 ### Standalone DogStatsD
 
-The default images run a dogstatsd server as well as the agent. If you want to run DogStatsD only, we provide [standalone images](https://github.com/DataDog/docker-dd-agent/tree/master/dogstatsd) for it. They are identified by the `dogstatsd` keyword in their docker tag (eg: `11.3.585-dogstatsd`, `11.3.585-dogstatsd-alpine`).
+The default images run a dogstatsd server as well as the main Agent (i.e. the collector). If you want to run DogStatsD only, we provide [standalone images](https://github.com/DataDog/docker-dd-agent/tree/master/dogstatsd) that don't run the collector. These images contain `dogstatsd` in their docker tag (e.g. `11.3.585-dogstatsd`, `11.3.585-dogstatsd-alpine`).
```

Comment:
@hkaj  Should we put forward `latest-dogstatsd` instead of a specific version? Else, we should update to the latest release at the time of merging